### PR TITLE
Add third party licenses file for release v0.5.0.

### PR DESCRIPTION
**Pull request by sondavidb**
**Issue #, if available:**

**Description of changes:**
Add third party licenses file for release v0.5.0.

**Testing performed:**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by Kern-- — APPROVED** [review]

**Review by turan18 — APPROVED** [review]

**Review by Kern-- — APPROVED** [review]

File: docs/getting-started.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ directory (`/usr/local/bin`):
 > You can find other download link in the release page that matches your machine.
 
 ```shell
-version="0.4.1"
+version="0.5.0"
```
Comment (dims) [inline docs/getting-started.md:57]: unrelated change for this PR? :)

Comment (sondavidb) [reply docs/getting-started.md:57]: This is intentionally part of the release process. We should probably make the commit message clearer to reflect this, though, or else make it a separate commit.
